templates/frontend/components/skipLinks.tpl

```diff
@@ -7,19 +7,18 @@
  *
  * Skip links to aid navigation when tabbing for screen reader accessibility
  *}
-<div class="cmp_skip_to_content">
-	<a href="#pkp_content_main">{translate key="navigation.skip.main"}</a>
-	<a href="#navigationPrimary">{translate key="navigation.skip.nav"}</a>
-	{if !$requestedPage || $requestedPage === 'index'}
-		{if $activeTheme && $activeTheme->getOption('showDescriptionInJournalIndex')}
-			<a href="#homepageAbout">{translate key="navigation.skip.about"}</a>
-		{/if}
-		{if $numAnnouncementsHomepage && $announcements|@count}
-			<a href="#homepageAnnouncements">{translate key="navigation.skip.announcements"}</a>
-		{/if}
-		{if $issue}
-			<a href="#homepageIssue">{translate key="navigation.skip.issue"}</a>
-		{/if}
+
+<a href="#pkp_content_main">{translate key="navigation.skip.main"}</a>
+<a href="#navigationPrimary">{translate key="navigation.skip.nav"}</a>
+{if !$requestedPage || $requestedPage === 'index'}
+	{if $activeTheme && $activeTheme->getOption('showDescriptionInJournalIndex')}
+		<a href="#homepageAbout">{translate key="navigation.skip.about"}</a>
 	{/if}
-	<a href="#pkp_content_footer">{translate key="navigation.skip.footer"}</a>
-</div>
+	{if $numAnnouncementsHomepage && $announcements|@count}
+		<a href="#homepageAnnouncements">{translate key="navigation.skip.announcements"}</a>
+	{/if}
+	{if $issue}
+		<a href="#homepageIssue">{translate key="navigation.skip.issue"}</a>
+	{/if}
+{/if}
+<a href="#pkp_content_footer">{translate key="navigation.skip.footer"}</a>
```

Comment:
The `<nav>` wrapper element should be placed here inside of the `skipLinks.tpl` file. See comment here: https://github.com/pkp/pkp-lib/pull/5343/files#r356657148